packages/multichain-provider/jest.config.js

```diff
@@ -0,0 +1,19 @@
+const deepmerge = require('deepmerge');
+const baseConfig = require('../../jest.config.base');
+
+module.exports = deepmerge(baseConfig, {
+  coveragePathIgnorePatterns: ['./src/Provider.ts'],
```

Comment:
Why?

Reply:
It's just types, and somehow not covered otherwise.